src/Identity/Controllers/SsoController.cs

```diff
@@ -132,14 +144,19 @@ public async Task<IActionResult> Login(string returnUrl)
                     { "domain_hint", domainHint },
                     { "scheme", scheme },
                 },
+                Parameters =
+                {
+                    { "ssoToken", ssoToken }
+                }
             };
 
             if (!string.IsNullOrWhiteSpace(userIdentifier))
             {
                 props.Items.Add("user_identifier", userIdentifier);
             }
 
-            return Challenge(props, scheme);
+            var challenge = Challenge(props, scheme);
```

Comment:
Please revert to original pattern. We can propose any changes to code-style patterns used throughout the code in a PIQ RFD.